test/edu/ucsb/nceas/metacat/MetacatHandlerTest.java

```diff
@@ -0,0 +1,1793 @@
+        dataStream = new FileInputStream(test_eml_essdive_file);
+        handler.save(newSys, true, MetacatHandler.Action.UPDATE,
+                     eml_format, dataStream, sysmeta, user);
+        // Check the objects
+        readSys = SystemMetadataManager.getInstance().get(pid);
+        assertTrue("The pid of systemmeta from db should be " + pid.getValue() + " rather than "
+                              + readSys.getIdentifier().getValue(),
+                              pid.getValue().equals(readSys.getIdentifier().getValue()));
+        assertTrue("The obsoletedBy of systemmeta from db should be " + newPid.getValue(),
+                        newPid.getValue().equals(readSys.getObsoletedBy().getValue()));
+        localId = IdentifierManager.getInstance().getLocalId(pid.getValue());
+        readObj = handler.read(localId, DocumentImpl.BIN);
+        readChecksum = getChecksum(readObj, MD5);
+        assertTrue("The read object should have the checksum " + test_eml_file_checksum
+                    + " rather than " + readChecksum, test_eml_file_checksum.equals(readChecksum));
+        assertNotEquals("The modification time of the obsoleted object in should change.",
+                        originModificationDate, readSys.getDateSysMetadataModified().getTime());
+        assertEquals("The upload time of the obsoleted object should not change.",
+                        originUploadDate, readSys.getDateUploaded().getTime());
+        // Newpid object exists.
+        SystemMetadata newReadSys = SystemMetadataManager.getInstance().get(newPid);
+        assertTrue("The pid of systemmeta from db should be " + newPid.getValue() + " rather than "
+                        + newReadSys.getIdentifier().getValue(),
+                        newPid.getValue().equals(newReadSys.getIdentifier().getValue()));
+        assertTrue("The obsoletes of systemmeta from db should be " + pid.getValue(),
+                        pid.getValue().equals(newReadSys.getObsoletes().getValue()));
+        assertNotEquals("The modification time in system metadata should change.",
+                            originModificationDateOfSecondObj,
+                            newReadSys.getDateSysMetadataModified().getTime());
+        assertEquals("The upload time in system metadata should not change.",
+                originUploadDateOfSecondObj, newReadSys.getDateUploaded().getTime());
+        localId = IdentifierManager.getInstance().getLocalId(newPid.getValue());
+        readObj = handler.read(localId, DocumentImpl.BIN);
+        readChecksum = getChecksum(readObj, MD5);
+        assertTrue("The object should have the checksum " + anotherChecksumStr + " rather than "
+                    + readChecksum, anotherChecksumStr.equals(readChecksum));
+    }
+
+    /**
+     * Test the save method to prevent overwriting an existing object
+     * @throws Exception
+     */
+    @Test
+    public void testSaveMethodWithMock() throws Exception {
```

Comment:
The console output for this test method starts with:

```log
LeanTestUtils: 'test.printdebug' is TRUE
LeanTestUtils: PropertyService initialized in UNIT_TEST mode; using default properties from lib/metacat.properties overlaid with configurable properties from test/test.properties
LeanTestUtils: PropertyService initialized in LIVE_TEST mode; using default properties from /opt/local/share/java/tomcat/webapps/metacat/WEB-INF/metacat.properties overlaid with configurable properties from /var/metacat/config/metacat-site.properties
```

The call to initialize `UNIT_TEST` is set up in your `setUp()` method, but I can't see where you are making a call to initialize `LIVE_TEST` in this method. I'm guessing it is not intentional?

Reply:
found it - line 1126. `D1NodeServiceTest` extends `MCTestCase`, which does this `LIVE_TEST` init.